content/solutions/java.html.haml

```diff
@@ -37,3 +37,39 @@ plugins:
   - 'Repository Connector plugin'
   - 'adds features for resolving artifacts from a Maven repository such as Nexus or Artifactory.'
 ---
+
+:markdown
+Jenkins supports building Java projects since its inception, and for a reason!
+It's both the language Jenkins is written in, plus the language in use by many if not all the
+projects Kohsuke Kawaguchi wanted to watch out when he created the tool many years ago.
+
+If you want to build a Java project, there are a bunch of different options.
+The most typical ones are generally Apache Maven, Gradle, Ant, or by custom scripts.
+
+## Apache Maven
+In any FreeStyle job, as *currently* Maven is supported in standard, you can use the
+dedicated step. One advantage is, as for all Jenkins tools, that you can select a specific
+Maven version and have Jenkins automatically install it on the build node it's gonna run on.
+
+%center
+  %img{:src => '/images/solution-images/jenkins-maven-step.png'}/
+
+## Gradle
+
+As the associated plugin is not installed per defaut, first install the
+
+%center
+%img{:src => '/images/solution-images/jenkins-gradle-step.png'}/
+
+## Using Ant
```

Comment:
As far as I'm aware, using Ant is definitely not "best practice" anymore. Android was the last big project using Ant that i was aware of, and they have since switched to Gradle.

I think we should stick to talking about Gradle and Maven with regards to Java projects.